R/utilities-entity.R

```diff
@@ -100,8 +100,11 @@ getAllEntitiesMatching <- function(paths, container, entityType) {
     stop(messages$errorWrongType("entityType", className, names(ContainerTasks)))
   }
 
+  task <- getContainerTask()
+  method <-  ContainerTasks[[className]]
+
   findEntitiesByPath <- function(path) {
-    toObjectType(rClr::clrCall(getContainerTask(), ContainerTasks[[className]], container$ref, path), entityType)

```

Comment:
Same here. This was retrieving the task object and variable everytime. Leveraging closure, it's only done once